docs/sdkx_android/tracking.mdx

```diff
@@ -89,7 +89,14 @@ Helpshift.showConversation(MainActivity.this, config);
 
 ## Attaching Custom Issue Fields to conversations {#set-custom-issue-fields}
 
-You can attach Custom Issue Fields to every new conversation started by the user. A Custom Issue Field should have a key, a datatype, and a value. The Helpshift SDK allows the addition of Custom Issue Fields by using the `customIssueFields` key in API config map. These Custom Issue Fields would be sent whenever a new conversation is started by the end user.
+You can attach Custom Issue Fields to every new conversation started by the user. A Custom Issue Field should have a key, a datatype, and a value. The Helpshift SDK allows the addition of Custom Issue Fields by using the `cifs` key in API config map.
+
+<Admonition type="info" title="Note">
+  We are deprecating the use of the "customIssueFields" key and introducing a
```

Comment:
We have deprecated...and introduced a new key....

Reply:
We would like to inform you that the usage of the `customIssueFields` key will be deprecated. We strongly recommend transitioning to using the `cifs` key as the preferred method for passing custom issue fields.